plugins/modules/net_tools/nmcli.py

```diff
@@ -1201,6 +1218,17 @@ def is_connection_changed(self):
         options.update(self.connection_options(detect_change=True))
         return self._compare_conn_params(self.show_connection(), options)
 
+    @staticmethod
+    def wifi_sec_secret_options():
```

Comment:
Why not have this is a constant, to avoid the function call and list creation every time a key is checked whether it's inside this list?

Reply:
Sure, that's not a bad idea, is this what you had in mind?

31bc0002b79dff755c2dedc87348b456fc9f48e1

Reply:
Yes, looks good, thanks!